vortexasdk/endpoints/voyages_geography_breakdown.py

```diff
@@ -114,6 +114,9 @@ def search(
         time_charterer: Union[ID, List[ID]] = None,
         time_charterer_excluded: Union[ID, List[ID]] = None,
         intra_movements: str = None,
+        voyage_date_range_activity: str = None,
```

Comment:
Same type issue as above, can you also fix the other occurrences of it?

Reply:
Changed to use a enum instead